src/lib/ngsi9/UpdateContextAvailabilitySubscriptionResponse.cpp

```diff
@@ -81,7 +81,7 @@ std::string UpdateContextAvailabilitySubscriptionResponse::render(void)
   out += startTag();
 
   out += subscriptionId.render(RtUpdateContextAvailabilitySubscriptionResponse, errorCodeRendered || durationRendered);
-  out += duration.render(false);
+  out += duration.render(errorCodeRendered);
```

Comment:
This could be a bugfixing... however, given it have nevered happened (in .test, etc.) and it is related with a very very very minor functionality (updating availability subscriptions... nobody uses that :) I think it doesn't worth the time further investigation.

NTC

Reply:
Sure, NTC